recipes/cubicinterpolation/all/conanfile.py

```diff
@@ -39,7 +39,7 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
-        self.requires("boost/1.75.0")
+        self.requires("boost/1.72.0")
```

Comment:
Can we go back up again?
```suggestion
        self.requires("boost/1.72.0")
```
```suggestion
        self.requires("boost/1.77.0")
```

Reply:
The problem is that we have users who would like to use the library on older Linux kernels (3.x).
However, boost filesystem is not compatible with Linux kernels <4.11 (see the [release notes](https://www.boost.org/doc/libs/1_75_0/libs/filesystem/doc/release_history.html)). That's why we wanted to use a boost version that still supports older systems.

Reply:
Those users can override the boost version in their `conanfile.txt` or profile.

Reply:
Oh, I didn't know about that.
Is it `-require-override`?

Reply:
I don't know how to use `--require-override` (but would love to know).

I was talking about using the following `conanfile.txt`:
```
[requires]
cubicinterpolation/0.1.4
boost/1.72.0
[generators]
cmake
cmake_find_package
```

Reply:
Looking [at this commit adding `--require-override`](https://github.com/conan-io/conan/commit/ae7732bea1ff21ee4f53e8c636558d6425a6b999), it only tests increasing the version of the requirement, not lowering it.

Reply:
And in my profile, I would have to change `build_requires`, correct?

Reply:
Only if boost is also used by some build requirement.

Reply:
I think recent boost can be used on older linux releases, by building it on a system with a sufficient low enough glibc.

Reply:
But if I see correctly, this can't be set with conan yet?

Reply:
glibc version is not part of the package id, currently.

Reply:
I've opened https://github.com/conan-io/conan/issues/10010.
I think it's a bug: conanfile.txt recipes has a different behavior then conanfile.py ones.